elytron/src/main/resources/schema/wildfly-elytron_10_0.xsd

```diff
@@ -1136,6 +1137,32 @@
 		</xs:complexContent>
 	</xs:complexType>
 
+    <xs:complexType name="failoverRealmType">
+        <xs:annotation>
+            <xs:documentation>
+                A realm definition which wraps one realm and delegates to another in case the first is unavailable.
```

Comment:
The description seems confusing to me a bit since the wrapped realm is delegate-realm and delegated realm is failover-realm.